packages/protocol/scripts/bash/release-on-devchain.sh

```diff
@@ -32,9 +32,19 @@ then
     BUILD_DIR=$(echo build/$(echo $BRANCH | sed -e 's/\//_/g'))
 fi
 
+
+rm -rf build/contracts*
+cd ../..
+yarn run reset
+yarn install >> $LOG_FILE
+yarn build >> $LOG_FILE
+cd packages/protocol
+
 echo "- Run local network"
 yarn devchain run-tar-in-bg packages/protocol/$BUILD_DIR/devchain.tar.gz >> $LOG_FILE
 
+# sleep 60
```

Comment:
does it actually sleep here?

Reply:
I left there to have it handy in case the CI didn't work without it, removing